src/scripts/main.js

```diff
@@ -8,10 +8,10 @@ window.addEventListener('hashchange', () => {
   }
 });
 
-const form = document.getElementById('form');
+// const form = document.getElementById('form');
 
-form.addEventListener('submit', (event) => {
-  event.preventDefault();
-  form.reset();
-  window.scrollTo(0, 0);
-});
+// form.addEventListener('submit', (event) => {
+//   // event.preventDefault();
+//   form.reset();
+//   window.scrollTo(0, 0);
+// });
```

Comment:
try filling in the form and submit it 🙂 
🎉 `Hello world` ???
that's a final thing to fix